pkg/scalers/externalscaler/externalscaler_grpc.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go-grpc. DO NOT EDIT.
 // versions:
 // - protoc-gen-go-grpc v1.2.0
-// - protoc             v3.21.12
+// - protoc             v3.6.1
```

Comment:
please remove this from the PR, it is not relevant and we are downgrading the protoc version